test/journeys/lib/test-helpers/space-widget/roster.js

```diff
@@ -22,3 +28,60 @@ export function hasParticipants(aBrowser, participants) {
   const participantsText = aBrowser.element(elements.rosterList).getText();
   return participants.map((participant) => assert.isTrue(participantsText.includes(participant.displayName)));
 }
+
+/**
+ * Verifies that a user can open the participant search feature
+ *
+ * @export
+ * @param {Object} aBrowser
+ * @returns {null}
+ */
+export function canSearchForParticipants(aBrowser) {
+  openSearch(aBrowser);
+  closeSearch(aBrowser);
+}
+
+/**
+ * Searches for a person and verifies that there are results
+ *
+ * @export
+ * @param {Object} aBrowser
+ * @param {String} searchString
+ * @param {boolean} doAdd actually add the person to participants
+ * @param {String} searchResult the string that should appear in search results
+ * @returns {null}
+ */
+export function searchForPerson(aBrowser, searchString, doAdd = false, searchResult = searchString) {
+  openSearch(aBrowser);
+  aBrowser.setValue(elements.searchInput, searchString);
+  aBrowser.waitForVisible(elements.addParticipantResultsArea);
+  aBrowser.waitForVisible(elements.addParticipantResultItem);
+  const resultsText = aBrowser.element(elements.addParticipantResultItem).getText();
+  assert.isTrue(resultsText.includes(searchResult), `matching search result is not found in results`);
```

Comment:
@moriahmaney having these messages with the `assert` makes the error log SO much nicer!